projects/1.18/assets/mythicbotany/mythicbotany/lang/zh_cn.json

```
@@ -1,181 +1,132 @@
+    "block.mythicbotany.exoblaze_floating": "浮空烈焰花",
+    "block.mythicbotany.wither_aconite": "凋零菟葵",
+    "block.mythicbotany.wither_aconite_floating": "浮空凋零菟葵",
+    "block.mythicbotany.aquapanthus": "百水莲",
+    "block.mythicbotany.aquapanthus_floating": "浮空百水莲",
+    "block.mythicbotany.hellebore": "鹿食草",
+    "block.mythicbotany.hellebore_floating": "浮空鹿食草",
+    "block.mythicbotany.raindeletia": "雨德花",
+    "block.mythicbotany.raindeletia_floating": "浮空雨德花",
+    "block.mythicbotany.feysythia": "Feysythia",
+    "block.mythicbotany.feysythia_floating": "浮空Feysythia",
+    "block.mythicbotany.petrunia": "符文矮牵牛",
+    "block.mythicbotany.petrunia_floating": "浮空符文矮牵牛",
+    "block.mythicbotany.yggdrasil_branch": "世界树的枝干",
+    "block.mythicbotany.rune_holder": "符文支架",
+    "block.mythicbotany.central_rune_holder": "大师符文支架",
+    "block.mythicbotany.mjoellnir": "雷神之锤",
+    "block.mythicbotany.dreamwood_leaves": "梦之木树叶",
+    "block.mythicbotany.elementium_ore": "源质钢矿石",
+    "block.mythicbotany.dragonstone_ore": "龙石矿石",
+    "block.mythicbotany.gold_ore": "金矿石",
+    "block.mythicbotany.return_portal": "亚尔夫海姆返回传送门",
+    "block.mythicbotany.raw_elementium_block": "粗源质钢方块",
+
+    "item.mythicbotany.alfsteel_helmet.description": "更远的攻击距离",
+    "item.mythicbotany.alfsteel_chestplate.description": "更强的抗击退强度",
+    "item.mythicbotany.alfsteel_leggings.description": "更快的移动速度",
+    "item.mythicbotany.alfsteel_boots.description": "更高的跳跃能力",
+
+    "block.mythicbotany.exoblaze.description": "烈焰!",
+    "block.mythicbotany.wither_aconite.description": "它需要吞噬下界之星",
+    "block.mythicbotany.aquapanthus.description": "天干物燥之时，试一试这个？",
+    "block.mythicbotany.hellebore.description": "如赫尔海姆般",
+    "block.mythicbotany.raindeletia.description": "雨（Rain）并不是苦（Pain）",
+    "block.mythicbotany.feysythia.description": "A Fairytale of Fey",
+    "block.mythicbotany.petrunia.description": "从女贞路的花园获得",
+
+    "enchantment.mythicbotany.hammer_mobility": "快速挥锤",
+    "enchantment.mythicbotany.hammer_mobility.desc": "增加锤子的攻击速度。",
+    "entity.mythicbotany.mjoellnir": "雷神之锤",
+    "entity.mythicbotany.alf_pixie": "亚尔夫海姆精灵",
+    "biome.mythicbotany.alfheim_plains": "精灵平原",
+    "biome.mythicbotany.alfheim_hills": "精灵山丘",
+    "biome.mythicbotany.dreamwood_forest": "梦之木森林",
+    "biome.mythicbotany.golden_fields": "黄金田野",
+    "biome.mythicbotany.alfheim_lakes": "精灵之湖",
+
+    "tooltip.mythicbotany.cost_pools": "%s 魔力池",
+    "tooltip.mythicbotany.reverse_ratio": "%sx 缩放 (悬停以放大)",
+    "tooltip.mythicbotany.rune_ritual": "符文仪式",
+    "tooltip.mythicbotany.rune_central": "符文需要大师符文支架",
+    "tooltip.mythicbotany.rune_consume": "这个符文会被消耗。",
+    "tooltip.mythicbotany.rune_keep": "这个符文不会被消耗。",
+    "tooltip.mythicbotany.rune_offset": "对应坐标: %s, %s",
+    "tooltip.mythicbotany.sacrifice_entity1": "实体必须靠近仪式。",
+    "tooltip.mythicbotany.sacrifice_entity2": "在这个过程中将被牺牲。",
+
+    "message.mythicbotany.mimir_knowledge": "喝下密米尔之泉水后，你就会获得最荒诞的梦境都难以设想的神秘知识。",
```

~我发现我把从句搞混了，应该改的其实是这个~
```suggestion
    "message.mythicbotany.mimir_knowledge": "喝下密米尔之泉水后，你获得了最荒诞的梦境都难以设想的神秘知识。",
```